modules/claims_api/spec/support/certificates/key.pem

```diff
@@ -0,0 +1,28 @@
+-----BEGIN PRIVATE KEY-----
+MIIEvQIBADANBgkqhkiG9w0BAQEFAASCBKcwggSjAgEAAoIBAQCwrmcF7zLDn5Td
+Y/xNaCxrIvIsGBF9F1MQvfiNzu1K5LrUOa0GfVGl+Oc0PK74aTzcL2maNhjY/TjX
+W5ctmx54QkRPFARcQnX33hs0a7rS1IrXBoex1lyOf3qw3GP/0W4fd2weC228r73J
+AxdHc8K8sNBurZSUJozJoqvmllUshK3PxanpNtkhlp07eYUryRwsB0ray7lFrYvl
+J/RwSj/M6/SXFm7IdOAtK9xS970+/SKfhGZkYw9Ch5K2/NlZshYagaDOpcIyDTCl
+4sl2pOT0KgZybmqCNZLx8oa980paLGUPtN9TECR5spZe58is8z/Abv4NnaS6ppJa
+wRy+o3fxAgMBAAECggEABAFkLQaD9lrhD1bTsraMoT4o0taV2OnlNHE2GBLrnsHK
+OYL4OPgw2Gwo466HgfKy0kQR5xDYZdzxw7IcHojQe0X3Ufqgca4BCiZDJCBYGyz9
+28uWWnIDZaiesbFBZByNnFGDBjhYm8k5KnwzNCe777ardwLg5i3sjaquLntUgX6T
+4pQbn0X2yKm3c6iXCm88hjzmDA5gSoVCo0QhA95R23iOvHc7h3/+josQ5Ezx8ZLR
+xkDNedvVIMZAlYSpGNRa72wxuT2tkA8oQlvn0zhPYbct0BB4J99x8Ye0QVYKyWVX
+lMKML+FK3yFD3g5ZYGTt/zqoEVGUso4bwzbgv5V3AQKBgQDdZbNXZXwDv6s2Ng3U
+5j1qSlv1TqGJAjXWdLMn+nB9/rtWp7egPt27rKbE12CRGxN2HKfXEKau5t/1Xhqw
+vPC/NXJZCtcatNb3xhWv5tBJmnQW0hB2cLJvwwF9BW/RM75eeH0a6lV33Bnt8VeY
+rtTPjhydEZdkeISw3o2XfboNMQKBgQDMS5M2wEeYfpCfK6ozTeXREqjqd/m2Hj2+
+kRrZ0He4c3GVL8YwlMk65R++0AEY1s4AlTLYLwZtFw4z5NaCIDAgyWH1+ZU5j8hd
+6qvFPZ5QTs1SRXiqZIuRolwIStWOAnBxJJIbQzawgXy+xajAmXvje71fDOJzwPAo
+2W+0Nn5mwQKBgQC9BiGykpDpWnHBXTE9/j2dTFNrA3FjWqHAse5sSaRVznEAMT3T
+8Ri7LVNNaF2oRie9hgW+1xaY6cF46pl7IDWj6uvwHhM9V/zZmyPVBQn8bYSEH2Gy
+3dFbHnFc3pFzWexFGDYoDr5eEWuJ/a4+S7aiZsEDUYuiWktf/wOAvfVlYQKBgFjn
+aqEg/Ac4zOb3sZGyDJj3bdrbayf98RDFRNIVDoSOeJdbuvvZQ9kFbNM04cVVdUB7
+rSy3CizfI6chptfp+gfKm37XpkDueVuljCrBret035U19lTwfGxcI4DYUCcEmLok
+GBEVJu+Y6hh7G/z0zh0o9Dgw+zs0c07TkPDz5f4BAoGAUaaT3mNU/5ijLVXua+Z/
+gNsoC/orbmDENsspSvjnq0w/FZ8wmQQRlsZfZ1MaWfOk2mEAwxurEEXM2ePw3lb8
+IfQxXsIc7rjodGBcmWxjJJWfskkEB7EplwofUO7a0O+1aOR/cxUgpC8xxKQucnVc
+IzikWGLSoRipcqi4Qlhu6EQ=
+-----END PRIVATE KEY-----
```

Comment:
assuming this is not a real / active key? and I don't really like the idea of pushing this through. I'm actually surprised that a scanning tool didn't alert on something like this 👀 . I'd prefer we mock or stub the call to get a token so we don't have to include this.

Reply:
@FonzMP The test suite kept failing because it was trying to open a .pem file.  I tried to get around it but creating this key and using it to be read in was the only solution I could find.  I only did that because I saw it being done in other spots like [these](https://github.com/department-of-veterans-affairs/vets-api/tree/master/spec/fixtures/sign_in) for example.  If there is a way around that totally all for it, just couldn't get anything else to run correctly.